package.json

```diff
@@ -76,12 +76,7 @@
   },
   "babel": {
     "presets": [
-      [
-        "es2015",
-        {
-          "loose": true
-        }
-      ]
+      ["env"]
```

Comment:
Isn't it better to use the current code instead of always the previous version?
So replacing `["env"]` with `"./src"`?